src/main/java/com/minecolonies/coremod/entity/EntityCitizen.java

```diff
@@ -1681,11 +1687,11 @@ private void tryPickupEntityItem(@NotNull final EntityItem entityItem)
             if (ItemStackUtils.isEmpty(resultStack) || ItemStackUtils.compareItemStacksIgnoreStackSize(itemStack, resultStack))
             {
                 CompatibilityUtils.getWorld(this).playSound((EntityPlayer) null,
-                        this.getPosition(),
-                        SoundEvents.ENTITY_ITEM_PICKUP,
-                        SoundCategory.AMBIENT,
-                        0.2F,
-                        (float) ((this.rand.nextGaussian() * 0.7D + 1.0D) * 2.0D));
+                  this.getPosition(),
+                  SoundEvents.ENTITY_ITEM_PICKUP,
+                  SoundCategory.AMBIENT,
+                  0.2F,
+                  (float) ((this.rand.nextGaussian() * 0.7D + 1.0D) * 2.0D));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 0.7D to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)